src/deadline_test_fixtures/deadline/worker.py

```diff
@@ -432,6 +440,9 @@ def _start_worker_agent(self) -> None:
         assert cmd_result.exit_code == 0, f"Failed to start Worker agent: {cmd_result}"
         LOG.info("Successfully started Worker agent")
 
+        # Wait 20 seconds for the worker to start before attempting to get ID
+        time.sleep(20)
+
         self.worker_id = self.get_worker_id()
```

Comment:
Likewise, here are we able to loop calling `get_worker_id` until it succeeds so that we can break out earlier if the worker starts in say 5 seconds

Reply:
Updated